crates/typst/src/layout/layout.rs

```diff
@@ -27,11 +27,11 @@ use crate::syntax::Span;
 /// ])
 /// ```
 ///
-/// If the `layout` call is placed inside of a box width a width of `{800pt}`
-/// and a height of `{400pt}`, then the specified function will be given the
-/// parameter `{(width: 800pt, height: 400pt)}`. If it placed directly into the
-/// page it receives the page's dimensions minus its margins. This is mostly
-/// useful in combination with [measurement]($measure).
+/// If the `layout` call is placed inside a box with a width of `{800pt}` and a
+/// height of `{400pt}`, then the specified function will be given the parameter
```

Comment:
```suggestion
/// height of `{400pt}`, then the specified function will be given the argument
```
I think generally, `parameter` is the variable at the def-site and `argument` is the actual value at the call-site, so I think this makes more sense.